features/scaffold-plugin-tests.feature

```diff
@@ -42,8 +42,34 @@ Feature: Scaffold plugin unit tests
     And the {PLUGIN_DIR}/hello-world/.travis.yml file should contain:
       """
       script:
-        - phpcs --standard=phpcs.ruleset.xml $(find . -name '*.php')
-        - phpunit
+        - |
+          if [[ ! -z "$WP_VERSION" ]] ; then
+            phpunit
+            WP_MULTISITE=1 phpunit
+          fi
+        - |
+          if [[ "$WP_TRAVISCI" == "phpcs" ]] ; then
+            phpcs --standard=phpcs.ruleset.xml $(find . -name '*.php')
+          fi
+      """
+    And the {PLUGIN_DIR}/hello-world/.travis.yml file should contain:
+      """
+      matrix:
+        include:
+          - php: 7.1
+            env: WP_VERSION=latest
+          - php: 7.0
+            env: WP_VERSION=latest
+          - php: 5.6
+            env: WP_VERSION=4.4
+          - php: 5.6
+            env: WP_VERSION=latest
+          - php: 5.6
+            env: WP_VERSION=trunk
+          - php: 5.6
+            env: WP_TRAVISCI=phpcs
```

Comment:
> Shouldn't this be run together with one of the other environments to save one iteration?

No. Breaking them out makes it easier to see whether `phpcs` failed or `phpunit` failed. Jobs are cheap; developer time is expensive.